test/dotnet.Tests/GivenThatTheUserIsRunningDotNetForTheFirstTime.cs

```diff
@@ -60,7 +60,7 @@ public void UsingDotnetForTheFirstTimeSucceeds()
         [Fact]
         public void UsingDotnetForTheFirstTimeWithNonVerbsDoesNotPrintEula()
         {
-            string firstTimeNonVerbUseMessage = Cli.Utils.LocalizableStrings.DotNetCommandLineTools;
+            const string firstTimeNonVerbUseMessage = $"{string.NewLine}.NET Core SDK (reflecting any global.json):";
```

Comment:
`error CS0117: 'string' does not contain a definition for 'NewLine'`

Did you mean Environment?

Reply:
Also this can't be const with string interpolation.